src/locales/ro/fields.json

```
@@ -100,6 +102,7 @@
   "Continental US": "SUA continentală",
   "Control points used to geolocate image, no terrain correction": "Puncte de control utilizate pentru geoetichetarea imaginii, fără corecție de teren",
   "Control points used to geolocate image, orthorectified": "Puncte de control utilizate pentru geoetichetarea imaginii, ortorectificate",
+  "Cookie": "",
```

There's something missing :-) Shall that be just Cookie or is there a translation in Romanian? @mneagul 

```suggestion
  "Cookie": "Cookie",
```